st3/sublime_lib/view_utils.py

```diff
@@ -117,18 +121,19 @@ def validate_view_options(options):
 
 
 def set_view_options(
-    view, *,
-    name=None,
-    settings=None,
-    read_only=None,
-    scratch=None,
-    overwrite=None,
-    syntax=None,
-    scope=None,
-    encoding=None,
-    content=None,
-    line_endings=None
-):
+    view: sublime.View,
+    *,
+    name: Optional[str] = None,
+    settings: Optional[dict] = None,
+    read_only: Optional[bool] = None,
+    scratch: Optional[bool] = None,
+    overwrite: Optional[bool] = None,
+    syntax: Optional[str] = None,
+    scope: Optional[str] = None,
+    encoding: Optional[str] = None,
+    content: Optional[str] = None,
+    line_endings: Optional[str] = None
```

Comment:
I'm not sure if we do this in the other places, but I prefer always including trailing commas in multi-line collections for cleaner diffs.

Reply:
Alas, Python doesn't allow a final trailing comma for keyword-only arguments. No idea why not, but it's a syntax error.

Reply:
Ah well, it's only been added in 3.6, sadly.